webapp/src/pages/Settings.tsx

```diff
@@ -31,48 +82,625 @@ const Settings: React.FC = () => {
+          Pipelines
+        </Typography>
+      </Box>
+      {resultingConfig.pipelines &&
+        _.sortBy(resultingConfig.pipelines, "name").map(
+          ({ name, model, postprocessors }, pipelineIndex) => (
+            <Box
+              key={pipelineIndex}
+              display="flex"
+              flexDirection="column"
+              margin={(theme) => theme.spacing(2, 3)}
+              padding={1}
+              border="1px solid rgba(0, 0, 0, 0.12)"
+            >
+              {displaySectionTitle("general")}
+              <Box
+                display="flex"
+                flexDirection="row"
+                justifyContent="flex-start"
+                marginLeft={2}
+              >
+                {displayReadonlyFields("name", name)}
+              </Box>
+              {displaySectionTitle("model")}
+              <Box
+                display="flex"
+                flexDirection="row"
+                justifyContent="flex-start"
+                alignContent="center"
+                flexWrap="wrap"
+                gap={(theme) => theme.spacing(1, 5)}
+                marginLeft={2}
+              >
+                {displayReadonlyFields("class_name", model.class_name)}
+                {displayReadonlyFields("remote", model.remote)}
+                {model.kwargs && displayArgumentsList("kwargs", model.kwargs)}
+                {model?.args &&
+                  model.args.length > 0 &&
+                  displayArgumentsList("args", model.args)}
+              </Box>
+              {displayPostprocessorToggleSection({
+                name,
+                model,
+                postprocessors,
+              })}
+              <Box
+                key={pipelineIndex}
+                display="flex"
+                flexDirection="column"
+                gap={1}
+                margin={2}
+              >
+                {postprocessors &&
+                  _.sortBy(postprocessors, "class_name").map(
+                    (postprocessor, index) => (
+                      <Box
+                        key={index}
+                        display="flex"
+                        flexDirection="row"
+                        justifyContent="flex-start"
+                        gap={2}
+                        padding={1}
+                        marginX={2}
+                        border="1px solid rgba(0, 0, 0, 0.12)"
+                      >
+                        {displayReadonlyFields(
+                          "class_name",
+                          postprocessor.class_name
+                        )}
+                        {postprocessor.temperature &&
+                          displayPostprocessorNumberField(
+                            { name, model, postprocessors },
+                            "temperature",
+                            index,
+                            postprocessor.temperature
+                          )}
+                        {postprocessor.threshold &&
```

Comment:
If you type 0 in the field, it disappears. You must check for `undefined`, not just any falsy value.
```suggestion
                        {postprocessor.temperature !== undefined &&
                          displayPostprocessorNumberField(
                            { name, model, postprocessors },
                            "temperature",
                            index,
                            postprocessor.temperature
                          )}
                        {postprocessor.threshold !== undefined &&
```

Reply:
Ohh, right!.. checking only undefined here is appropriate